From 5f68ea8a307b4e78adb8dd985cd9d2319f367a71 Mon Sep 17 00:00:00 2001
From: Justin Maggard <jmaggard@netgear.com>
Date: Wed, 26 Oct 2016 20:12:08 +0000
Subject: [PATCH 13/21] Revert "resolver: Try writing resolv.conf to STATEDIR,
 fall back to /etc"

This reverts commit 3a9ad49c8c8448875375a67913af98f74bca0ad7.
---
 src/resolver.c | 17 +++--------------
 1 file changed, 3 insertions(+), 14 deletions(-)

diff --git a/src/resolver.c b/src/resolver.c
index 75ea5ba..b637e87 100644
--- a/src/resolver.c
+++ b/src/resolver.c
@@ -35,9 +35,6 @@
 
 #include "connman.h"
 
-#define RESOLV_CONF_STATEDIR STATEDIR"/resolv.conf"
-#define RESOLV_CONF_ETC "/etc/resolv.conf"
-
 #define RESOLVER_FLAG_PUBLIC (1 << 0)
 
 /*
@@ -133,19 +130,11 @@ static int resolvfile_export(void)
 
 	old_umask = umask(022);
 
-	fd = open(RESOLV_CONF_STATEDIR, O_RDWR | O_CREAT | O_CLOEXEC,
+	fd = open("/etc/resolv.conf", O_RDWR | O_CREAT | O_CLOEXEC,
 					S_IRUSR | S_IWUSR | S_IRGRP | S_IROTH);
 	if (fd < 0) {
-		connman_warn_once("Cannot create "RESOLV_CONF_STATEDIR" "
-			"falling back to "RESOLV_CONF_ETC);
-
-		fd = open(RESOLV_CONF_ETC, O_RDWR | O_CREAT | O_CLOEXEC,
-					S_IRUSR | S_IWUSR | S_IRGRP | S_IROTH);
-
-		if (fd < 0) {
-			err = -errno;
-			goto done;
-		}
+		err = -errno;
+		goto done;
 	}
 
 	if (ftruncate(fd, 0) < 0) {
-- 
2.1.4

